.github/workflows/plugin.yaml

```diff
@@ -16,12 +16,6 @@ jobs:
         pex-tool:
           - default
           - in-repo
-        exclude:
-          # Disabled until a please_pex release works with Python >= 3.12:
-          - python-version: '3.12'
-            pex-tool: default
-          - python-version: '3.13'
-            pex-tool: default
```

Comment:
These were excluded in #204 - the new please_pex version should be compatible with Python >= 3.12 so they can be re-added.